src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -4803,7 +4811,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Next
             End If
 
-            Dim block = Me.BindBlock(node, node.Statements, diagnostics)
+            Dim block = Me.BindBlock(node, node.Statements, diagnostics).MakeCompilerGenerated()
```

Comment:
It looks like most `BoundBlock` instances are now marked compiler generated, but method body blocks are not. What is the difference with method body blocks?

Reply:
> What is the difference with method body blocks?

Their associated syntax doesn't conflict with other IOperation nodes. Also, GetOperation API returns only explicit IOperation nodes.